src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -315,5 +315,10 @@ public Task<ResourceWrapper> UpdateSearchIndexForResourceAsync(ResourceWrapper r
         {
             throw new NotImplementedException();
         }
+
+        public async Task<int?> GetProvisionedDataStoreCapacityAsync(CancellationToken cancellationToken = default)
```

Comment:
nit: This makes me think you could split out this interface from the IDataStore i.e. IProvidesProvisionedCapacity? Then this doesn't need to be implemented.